mp/game/momentum/momentum.fgd

```diff
@@ -7,6 +7,13 @@
 // -----------------------------------------------------------------------
 @SolidClass base(Targetname, EnableDisable, Toggle) = func_nogrenades : "Rockets and grenades will not detonate inside this area." 
 [
+	fizzle_type(choices) : "Stickybomb prevention method" : 0 =
+	[
+		0 : "Fizzle on detonation."
+		1 : "Fizzle when the sticky lands."

```

Comment:
Would be good to reference `airborne_only` here to explain it better.

Reply:
Would "Fizzle when the sticky lands. Prevent airborne explosions only must be true." be a better description?

Reply:
I'd say "Fizzle when the stickybomb lands if airborne_only is false."